twentynineteen-blocks/style.css

```diff
@@ -112,6 +120,27 @@ Tags: one-column, flexible-header, accessibility-ready, custom-colors, custom-me
   }
 }
 
+/* Latest posts block */
+.wp-block-latest-posts li > a {
```

Comment:
This is a great example of the things we should be absorbing in the block style work. cc @ItsJonQ @nosolosw.

Reply:
One potential answer for this would be this proposal https://github.com/WordPress/gutenberg/pull/20290